packages/wells/src/client/api/wellboresAPI.ts

```diff
@@ -0,0 +1,30 @@
+import { CDFHttpClient} from '@cognite/sdk-core';
+import { Survey } from '../model/Survey';
+
+export class WellboresAPI {
+  private client?: CDFHttpClient;
+  private project?: String;
+
+  public set setHttpClient(httpClient: CDFHttpClient) {
+    this.client = httpClient;
+  }
+
+  public set setProject(project: String) {
+    this.project = project;
+  }
+
+  /* eslint-disable */
+  public getTrajectory = async (wellboreId: number): Promise<Survey | undefined> => {
+      const path: string = `/${this.project}/wellbores/${wellboreId}/trajectory`
```

Comment:
should an error be returned if the project is not defined? I think the API will just return a 404, which wouldn't very helpful?

Reply:
I guess we can set a 400 Bad Request if the project is not set in the client?

Reply:
Sounds good!